### DISCUSSION TITLE
I want to remove the design option from the admin menu

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#12134] [Q&A] [answered]
Hi I wanted to know on how to remove the design option from menu. please find the snapshot showcasing the design option which I want to remove

![image](https://user-images.githubusercontent.com/25971751/182541784-2f724089-de4a-438c-8d4d-10b57fb4cdf2.png)

**Comment by ns8482e** [comment, marked as answer]
It’s permission driven, you need create a role that doesn’t have ManageTemplates/ManagePlacements permission  and assign that role to user

**Reply by syedshadab19** [reply]
@ns8482e  thanks for the reply, I tried it out but still appears can you please specify which permission to be removed to hide the design.

**Reply by ns8482e** [reply]
You need to remove all permissions

after removing Managetemplates what other menus you see?

**Reply by syedshadab19** [reply]
@ns8482e  thank you i was able to remove it using permission

**Comment by hishamco** [comment]
You could remove the menu item using `NavigationBuilder.Remove()`

**Reply by ns8482e** [reply]
Removing only from navigation will still allow those users to access feature provided if access is not removed

If goal is not to allow the user to perform such action then it should be handled using permission

**Reply by hishamco** [reply]
You are right
